server/graphql/v2/object/PaymentMethod.js

```diff
@@ -31,9 +33,16 @@ export const PaymentMethod = new GraphQLObjectType({
       },
       service: {
         type: GraphQLString,
+        deprecationReason: '2020-08-18: This field is being deprecated in favor of providerType',
       },
       type: {
         type: GraphQLString,
+        deprecationReason: '2020-08-18: This field is being deprecated in favor of providerType',
+      },
+      providerType: {
```

Comment:
maybe `paymentMethodType` instead if it's temporary

Reply:
I hesitated between the two (tho this field should really be called `type`) and went with `providerType` because it's not used in our codebase today (= easier to refactor) and is, in my opinion, pretty explicit about what it is: the type of the payment method provider.